### PR TITLE
Update timestamp names in database schema - Closes # 619

### LINKED ISSUE #619: Update timestamp names in database schema
### Description

Rename every occurence of `ctime` and `mtime` in the database schema.

`ctime` -> `createdAt`
`mtime` -> `modifiedAt`

Make sure the database requests are handled correctly after that change.

### Motivation

- `ctime`/`mtime` might be hard to distinguish by developers

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #619 

### How was it solved?
- [x] Rename ctime to createdAt
- [x] Rename mtime to modifiedAt
- [x] Support previous version `v1`

### How was it tested?
Local + Jenkins

### CODE REVIEW

File: services/newsfeed/shared/schema/newsfeed.js [hunk 1]
```diff
@@ -24,8 +24,8 @@ module.exports = {
 		image_url: { type: 'string' },
 		source: { type: 'string', null: false },
 		source_id: { type: 'string' },
-		ctime: { type: 'integer', null: false },
-		mtime: { type: 'integer' },
+		createdAt: { type: 'integer', null: false },
+		modifiedAt: { type: 'integer' },
```
Comment (sameersubudhi) [inline services/newsfeed/shared/schema/newsfeed.js:28]: Kindly consider setting `modifiedAt` to be `non-nullable`. (The values should be defaulting to the corresponding `createdAt` values)
If not, please consider updating the Joi schema for the newsfeed articles in the below files to be `optional()`:

- services/newsfeed/tests/schemas/newsfeedArticle.schema.js
- tests/schemas/newsfeed.schema.js

File: services/gateway/sources/version2/mappings/newsfeed.js [hunk 2]
```diff
@@ -19,8 +19,8 @@ module.exports = {
 	imageUrl: '=,string',
 	source: '=,string',
 	source_id: '=,string',
-	ctime: '=,number',
-	mtime: '=,number',
+	createdAt: '=,number',
```
Comment (sameersubudhi) [inline services/gateway/sources/version2/mappings/newsfeed.js:22]: Does `v2` also need to be backward compatible?

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
Kindly consider the following.

File: services/gateway/sources/version2/mappings/newsfeed.js (hunk 2 above)
Comment (nagdahimanshu) [reply services/gateway/sources/version2/mappings/newsfeed.js:22]: No, I had already discussed with @MichalTuleja and there is no need for it.

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Review by MichalTuleja — APPROVED** [review]

**Pull request merged**